app/src/main/java/com/example/firstandroid/presentation/update_bathroom/UpdateBathroomScreen.kt

```diff
@@ -0,0 +1,127 @@
+package com.example.firstandroid.presentation.update_bathroom
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+
+@Composable
+fun UpdateBathroomDialog(
+    onDismissRequest: () -> Unit,
+    viewModel: UpdateBathroomViewModel,
+    originalBathroomInfo: UpdateBathroomState,
+) {
+
+    val state = viewModel.state
+    val scope = rememberCoroutineScope()
```

Comment:
Instead of using all of these local state items, you could use the state held in the Viewmodel.